alembic/versions/5ca6efb386e8_add_owner_column_to_site_model.py

```diff
@@ -0,0 +1,28 @@
+"""Add Owner column to Site Model
+
+Revision ID: 5ca6efb386e8
+Revises: 81278a3571b2
+Create Date: 2025-03-11 22:19:21.689554
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5ca6efb386e8'
+down_revision = '81278a3571b2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('sites', sa.Column('owner', sa.String(length=255), nullable=True, comment='The owner of the site or data source'))
+    # ### end Alembic commands ###
+
```

Comment:
Sorry could this be changed from a string to a optional join with the user table. So the owner then is a person in the user table

Reply:
it might need to be owner, and owner_uuid as the way to join it